lib/client.js

```diff
@@ -50,7 +50,7 @@ export default (function INIT() {
 
           const dir = self.getProjectPath();
 
-          const watchDataFormatted = watchData.map(watch => Path.resolve(dir, watch));
+          const watchDataFormatted = watchData.map(watch => Path.resolve(dir+watch));
```

Comment:
This is strange.. it should not change the behavior... 
I'll test this, but can you elaborate on why Path.resolve did not work the way it was intended?
See https://nodejs.org/api/path.html#path_path_resolve_paths